source/grid/tria.cc

```diff
@@ -1689,7 +1689,7 @@ namespace internal
               default:
                 // in 1d, a node must have one or two adjacent lines
                 if (spacedim==1)
-                  AssertThrow (false, ExcInternalError())
+                  Assert (false, ExcInternalError())
```

Comment:
I checked after I wrote this: we define `Assert` as a curly bracket block, so this is syntactically okay: it does confuse the indentation script, though.